src/chatlog/chatlog.cpp

```diff
@@ -560,11 +562,19 @@ void ChatLog::clear()
 {
     clearSelection();
 
+    QVector<ChatLine::Ptr> savedLines;
+
     for (ChatLine::Ptr l : lines)
-        l->removeFromScene();
+        if (!isActiveFileTransfer(l))
```

Comment:
It would make more sense to remove the `!` and switch if and else parts. Removing the "not" removes one step of thinking -> easier :)